stable/karma/values.yaml

```diff
@@ -126,3 +126,6 @@ configMap:
   #   sentry:
   #     private: secret
   #   public: 123456789
+
+# Name of a secret that will be mounted to /etc/certs in the karma container.
+certSecretName: ""
```

Comment:
would it make sense to make this a list (as karma supports multiple alertmanagers) - or is this a bit over the top?

Reply:
`Mount a secret to `/etc/certs``, so I think I wouldn't really get that it's for alertmanager TLS config.
Yes, having ability to set only one seems a bit limiting, eventually someone will need more.

Reply:
👍 I'll update the parameter to take a list of secret names. I'll also clarify the language in the README to make clear that it's intended for Alertmanager TLS config.

Reply:
Fixes pushed. I tried to keep the description in the README short and crisp, and add a verbose description as a comment in `values.yaml`. Please let me know if you'd like the README entry to be more verbose.